docs/java/how-to-spawn-a-child-workflow-execution-in-java.md

```diff
@@ -0,0 +1,135 @@
+  }
+  // Child Workflow implementation not shown
+
+  // Parent Workflow implementation
+  public class GreetingWorkflowImpl implements GreetingWorkflow {
+
+  @Override
+  public String getGreeting(String name) {
+      GreetingChild child = Workflow.newChildWorkflowStub(GreetingChild.class);
+
+      // This is a blocking call that returns only after child has completed.
+      return child.composeGreeting("Hello", name );
+  }
+  }
+  ```
+
+- Spawn two Child Workflows (with the same type) in parallel:
+
+  ```java
+  // Parent Workflow implementation
+  public class GreetingWorkflowImpl implements GreetingWorkflow {
+
+      @Override
+      public String getGreeting(String name) {
+
+          // Workflows are stateful, so a new stub must be created for each new child.
+          GreetingChild child1 = Workflow.newChildWorkflowStub(GreetingChild.class);
+          Promise<String> greeting1 = Async.function(child1::composeGreeting, "Hello", name);
+
+          // Both children will run concurrently.
+          GreetingChild child2 = Workflow.newChildWorkflowStub(GreetingChild.class);
+          Promise<String> greeting2 = Async.function(child2::composeGreeting, "Bye", name);
+
+          // Do something else here.
+          ...
+          return "First: " + greeting1.get() + ", second: " + greeting2.get();
+      }
+  }
+  ```
+
+- Send a Signal to a Child Workflow from the parent:
+
+  ```java
+  // Child Workflow interface
+  @WorkflowInterface
+  public interface GreetingChild {
+      @WorkflowMethod
+      String composeGreeting(String greeting, String name);
+
+      @SignalMethod
+      void updateName(String name);
+  }
+
+  // Parent Workflow implementation
+  public class GreetingWorkflowImpl implements GreetingWorkflow {
+
+      @Override
+      public String getGreeting(String name) {
+          GreetingChild child = Workflow.newChildWorkflowStub(GreetingChild.class);
+          Promise<String> greeting = Async.function(child::composeGreeting, "Hello", name);
+          child.updateName("Temporal");
+          return greeting.get();
+      }
+  }
+  ```
+
+- Sending a Query to Child Workflows from within the parent Workflow code is not supported. However, you can send a Query to Child Workflows from Activities using `WorkflowClient`.
+
+### ParentClosePolicy
+
+When creating a Child Workflow, you can define a `ParentClosePolicy` that terminates, cancels, or abandons the Child Workflow Execution if the parent Workflow Execution stops.
+
+- `ABANDON`: When the parent stops, don't do anything with the Child Workflow.
+- `TERMINATE`: When the parent stops, terminate the Child Workflow
+- `REQUEST_CANCEL`: When the parent stops, terminate the Child Workflow
+
+You can set policies per Child Workflow, which means you can opt out of propagating terminations and cancellations on a per-child basis.
+This is useful for starting Child Workflows asynchronously, as shown in the following example:
```

Comment:
This whole section of text is conceptual explanation which is covered in the "What is a Parent Close Policy" page: https://docs.temporal.io/docs/concepts/what-is-a-parent-close-policy

Additionally - this actually maps to "How to set a Parent Close Policy in Java" - which is done via Child Workflow Options i.e. "How to set `ChildWorkflowOptions` in Java" - not that you have to do all this here - just a heads up & fyi.

Reply:
I'll do it. This is important.